ci/workflows/main.yml

```diff
@@ -0,0 +1,103 @@
+steps:
```

Comment:
Are steps running in parallel? I guess debug build, test, e2e should be separated into multiple workflows?

Reply:
Steps can run in parallel, of course, you can also set order dependencies.